packages/leavittbook/src/components/titanium-data-table-header.ts

```diff
@@ -0,0 +1,40 @@
+import { LitElement, html, css } from 'lit';
+import { customElement } from 'lit/decorators.js';
+import { h1, h2, h3, h5, p } from '@leavittsoftware/titanium-styles';
+import StoryStyles from '../styles/story-styles';
+import '@leavittsoftware/titanium-data-table/lib/titanium-data-table-header';
+import '@leavittsoftware/titanium-card';
+import '../shared/code-block';
+import '../shared/story-header';
+import '@api-viewer/docs';
+
+@customElement('titanium-data-table-header-demo')
+export class TitaniumDateTableHeaderDemoElement extends LitElement {
+  static styles = [h1, h2, h3, h5, p, StoryStyles, css``];
+
+  #defaultStory() {
+    return html`
+      <div>
+        <h1>Default</h1>
+        <p>Examples using large, right, no-sort, and width attributes</p>
+        <titanium-data-table-header large column-name="Name" large title="Name"></titanium-data-table-header>
+        <titanium-data-table-header column-name="SName" desktop title="Short Name"></titanium-data-table-header>
+        <titanium-data-table-header column-name="person_type" right desktop title="Type"></titanium-data-table-header>
+        <titanium-data-table-header no-sort desktop center width="105px" title="Phone Number"></titanium-data-table-header>
```

Comment:
Can we just do one property per header so it is clear what that property does?

Reply:
[68fe7c2](https://github.com/LeavittSoftware/titanium-elements/pull/336/commits/68fe7c20e33125a29cc362379f7baefa6ffd8a72)